html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <div>
+        <div class="Email">
+          <label for="Email">Email</label>
```

Comment:
You have the `E` capitalized here, but it is lowercase on the `id` for the `input` this is paired with.